lesson-02-cw721-soulbound/exercise-01/exercise.md

```diff
@@ -7,25 +7,23 @@ Title: Contracts and Libraries
 Filename: lib.rs
 -->
 
-> If Section 31 didn't like what you were doing, they could have just vapourized the teleporter (not to mention your entire DNA). So why'd you end up in their cyber jail? It's got to have something to do with this missing `passport-token`.
+> If Section 31 didn't like what you were doing, they could have just vapourized the teleporter (not to mention your entire DNA). So why'd you end up in their cyber jail? It must have something to do with the missing `passport-token`.
 
-It occurs to you that you may need to build an NFT just get out of here!
+It occurs to you that you may need to build an NFT just to get out of here!
 
-When it comes to making NFT collections with custom logic, there are a few strategies that could be followed.
+When it comes to making NFT collections with custom logic, there are a few strategies that could be followed, such as:
 
-For example:
-
-1. Add customizations in the token collection contract
-2. Bootstrap a version of `cw721-base` locally within our project and modify it
-3. Import modified versions of `cw721-base` as package dependencies
+1. Adding customizations in the token collection contract
+2. Bootstrapping a version of `cw721-base` locally within the project and modifying it
+3. Importing modified versions of `cw721-base` as package dependencies
```

Comment:
We should change number 3 to:
"Importing a modified version of `cw721-base` as a package dependency"